docs/contracts/inheritance.rst

```diff
@@ -484,7 +484,31 @@ inheritance list or in modifier-style in the derived constructor.
 Specifying arguments in both places is an error.
 
 If a derived contract does not specify the arguments to all of its base
-contracts' constructors, it will be abstract.
+contracts' constructors, it must be declared abstract. In that case, when
+a contract derives from it, its declaration or its constructor must
```

Comment:
This is a bit hard to parse because "contract" and "it" are used multiple times referring to different things. How about this?
```suggestion
another contract derives from it, that contract's declaration or constructor must
```